names.md

```diff
@@ -9,5 +9,5 @@ Shaurav Ranjan
 Vibha Yadav- vib
 Akansha Chitransh - Gunnie
 praveen raturi - lovey
-Hrishikesh - rishi
+Hrishikesh - rishi2
```

Comment:
You based your TR-11 off your color branch.  You should have based it off your master branch